src/content/docs/apm/reports/performance-reports.mdx

```diff
@@ -318,35 +314,43 @@ The scalability analysis report includes three charts.
       </td>
     </tr>
 
     <tr>
       <td>
-        Database utilization
+        Database
       </td>
 
       <td>
-        This chart plots your application's database utilization versus application load in terms of requests per minute. For an application that is scaling well, you should see a roughly linear pattern in the plotted points.
+        The chart in this tab plots your application's database time versus application load in terms of requests per minute. For an application that is scaling well, you should see a roughly linear pattern in the plotted points. The billboard shows the average, minimum, and maximum database times, and average throughput.
       </td>
     </tr>
 
     <tr>
       <td>
-        CPU utilization
+        APM CPU
       </td>
 
       <td>
-        This chart plots your application's CPU time per request for the Rails process, versus application load in terms of requests per minute. For an application that is scaling well, you should see a roughly linear pattern in the plotted points.
+        The chart in this tab plots your application's CPU time per request, versus application load in terms of requests per minute. For an application that is scaling well, you should see a roughly linear pattern in the plotted points. The billboard shows the average, minimum, and maximum CPU times, and average throughput.
       </td>
     </tr>
   </tbody>
 </table>
 
+The charts feature two lines: the primary line connects the data points, while the secondary line, labeled as "smoothed," presents a smoothed version of the primary line. The secondary line helps visualize the application's performance without being concerned about unexpected fluctuations observed in the primary line.
+
+These charts are constructed a bit differently from other New Relic charts and do not have the default chart actions commonly available.
+
 ## Select options for time periods [#time]
 
-Each scalability analysis report includes options to view the data for different time periods, including the last 24 hours, last 7 days, or other. For example, if you select **Use time window selection** and the [time picker](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#dash-time-picker) uses the default value (last 30 minutes, ending now), the chart shows only the data for the last 30 minutes.
+The Scalability report has a [time picker](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#dash-time-picker) which allows you to view data over different time periods. The minimum recommended time period is 30 minutes because when analyzing an application’s Scalability, lower time ranges don’t give a full picture. The time picker uses the default value of the last 30 minutes, ending now.
```

Comment:
```suggestion
The Scalability report has a [time picker](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#dash-time-picker) that allows you to view data over different time periods. The minimum recommended time period is 30 minutes because when analyzing an application’s scalability, lower time ranges don’t give a full picture. The time picker uses the default value of the last 30 minutes.
```